spec/services/shopify_import/products/create_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
 
-RSpec.describe Shopify::Products::Imports::Create, type: :service do
+RSpec.describe ShopifyImport::Products::Create, type: :service do
   let!(:client) { ShopifyAPI::Base.site = 'https://foo:baz@test_shop.myshopify.com/admin' }
```

Comment:
I know this is old code, but this is incorrect use of `let`. Basically here we're doing `before(:each)`, but calling it a variable `client` through `let!`. Also, this variable name would suggest this is a `ShopifyImport::Client`, while it's actually a return values of calling `ShopifyAPI::Base.site=` method (which happens to be the same string that's passed to the method)